app/javascript/admin/controllers/config_controller.js

```diff
@@ -110,21 +125,23 @@ export default class ConfigController extends Controller {
 
   activateEmailAuthModal(event) {
     event.preventDefault();
-    this.configModalAnchorTarget.innerHTML = adminModal(
-      emailAuthModalTitle,
-      emailAuthModalBody,
-      'Confirm disable',
-      'disableEmailAuthFromModal',
-      'Cancel',
-      'closeAdminConfigModal',
-    );
+    this.configModalAnchorTarget.innerHTML = adminModal({
+      title: emailAuthModalTitle,
+      body: emailAuthModalBody,
+      leftBtnText: 'Confirm disable',
+      leftBtnAction: 'disableEmailAuthFromModal',
+      rightBtnText: 'Cancel',
+      rightBtnAction: 'closeAdminModal',
+      leftBtnClasses: 'crayons-btn--danger',
+      rightBtnClasses: 'crayons-btn--secondary',
+    });
     this.positionModalOnPage();
   }
 
   disableEmailAuthFromModal(event) {
     event.preventDefault();
     this.disableEmailAuth(event);
-    this.closeAdminConfigModal(event);
+    this.closeAdminModal(event);
```

Comment:
refactor (blocking): `closeAdminModal` takes no parameters.

```suggestion
    this.closeAdminModal();
```